conf/application.conf

```diff
@@ -47,7 +47,7 @@ microservice {
       trader-goods-profiles-router {
         protocol = http
         host     = localhost
-        port     = 10903
+        port     = 10904
```

Comment:
I did this, too - we should probably all catch up about whether we point to router or stubs from here on out.

Reply:
agreed